src/components/Layout/index.tsx

```diff
@@ -9,6 +9,7 @@ import { Loader } from '../tasks/card/Loader';
 
 interface Props {
     children?: ReactNode;
+    hideHeader?: boolean; // Hides header when set to true
 }
 
 const navBarContent = (title: string, refUrl: string, isActive = false) => {
```

Comment:
- Please Change the first letter to a capital letter this is generally followed when building components in react.

Reply:
Done.